.github/workflows/pull-request.yaml

```diff
@@ -45,3 +45,23 @@ jobs:
         uses: actions/checkout@v4
       - name: Check differences
         run: make check-dashboard-updates
+
+  integration-test:
+    needs: [code-quality, unit-test]
+    uses: canonical/bootstack-actions/.github/workflows/func.yaml@v3
+    strategy:
+      fail-fast: false
+    with:
+      command: "tox -e integration"
+      juju-channel: "3.1/stable"
+      nested-containers: false
+      timeout-minutes: 120
+      action-operator: false
+      external-controller: true
+      runs-on: "['self-hosted', 'runner-openstack-exporter']"
+      juju-controller: soleng-ci-ctrl
```

Comment:
`ERROR "soleng-ci-ctrl" is not the name of a model or controller` @jneo8  can you help to check?